src/contracts/pods/EigenPod.sol

```diff
@@ -162,6 +162,12 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         _;
     }
 
+    modifier partialWithdrawalProofSwitchOn() {
```

Comment:
nit: `whenPartialWithdrawalProofSwitchOn` for better readability and clarification